source/code/plugins/filter_syslog.rb

```diff
@@ -30,6 +30,7 @@ def filter(tag, time, record)
       # Use Time.now, because it is the only way to get subsecond precision in version 0.12.
       # The time may be slightly in the future from the ingestion time.
       record["Timestamp"] = OMS::Common.format_time(Time.now.to_f)
+      record["EventTime"] = OMS::Common.format_time(time)
```

Comment:
Can you provide me access to the machines? I don't see how this could happen. The 'time' parameter comes from the raw syslog message (rounded to second) and is the unix/epoch time integer - so inherently UTC. So both, Time.now and 'time' are in UTC - I don't see how 'time' could be later/after Time.now.